File: src/main/java/fr/lewon/web/bot/entities/response/DraftResponse.java
package fr.lewon.web.bot.entities.response;

import java.util.List;

import com.fasterxml.jackson.annotation.JsonProperty;

import fr.lewon.web.bot.entities.girls.Girl;

public class DraftResponse extends Response {

	@JsonProperty("teamArray")
	private List<Girl> team;
	
	@JsonProperty
	private Boolean canDraft;

	@JsonProperty
	private Integer freeDrafts;

	@JsonProperty
	private Integer priceDraft;

	public List<Girl> getTeam() {
		return team;
	}

	public void setTeam(List<Girl> team) {
		this.team = team;
	}

	public Boolean getCanDraft() {
		return canDraft;
	}

	public void setCanDraft(Boolean canDraft) {
		this.canDraft = canDraft;
	}

	public Integer getFreeDrafts() {
		return freeDrafts;
	}

	public void setFreeDrafts(Integer freeDrafts) {
		this.freeDrafts = freeDrafts;
	}

	public Integer getPriceDraft() {
		return priceDraft;
	}

	public void setPriceDraft(Integer priceDraft) {
		this.priceDraft = priceDraft;
	}
	
}

File: src/main/kotlin/fr/lewon/bot/hh/entities/response/DraftResponse.kt
package fr.lewon.bot.hh.entities.response

import com.fasterxml.jackson.annotation.JsonProperty
import fr.lewon.bot.hh.entities.girls.Girl

class DraftResponse : Response() {
    @JsonProperty("teamArray")
    var team: List<Girl>? = null
    @JsonProperty
    var canDraft: Boolean? = null
    @JsonProperty
    var freeDrafts: Int? = null
    @JsonProperty
    var priceDraft: Int? = null

}